converter/src/test/java/gov/cms/qpp/acceptance/MultipleTinRoundTripTest.java

```diff
@@ -1,35 +1,38 @@
 package gov.cms.qpp.acceptance;
 
-import com.jayway.jsonpath.JsonPath;
-import com.jayway.jsonpath.ReadContext;
-import gov.cms.qpp.acceptance.helper.JsonPathToXpathHelper;
-import gov.cms.qpp.conversion.encode.JsonWrapper;
-import org.junit.BeforeClass;
-import org.junit.Ignore;
-import org.junit.Test;
+import static com.google.common.truth.Truth.assertThat;
 
 import java.io.IOException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.List;
 import java.util.Map;
 
-import static com.google.common.truth.Truth.assertThat;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
 
-@Ignore
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.ReadContext;
+
+import gov.cms.qpp.acceptance.helper.JsonPathToXpathHelper;
+import gov.cms.qpp.conversion.encode.JsonWrapper;
+
+@Disabled
```

Comment:
Do we have a story or task that will encourage us to do something with this so that the tests aren't ignored?

Reply:
Nope. I'll add it now.